eng/Versions.props

```diff
@@ -256,6 +256,9 @@
     <IdentityServer4StoragePackageVersion>3.0.0</IdentityServer4StoragePackageVersion>
     <IdentityServer4EntityFrameworkStoragePackageVersion>3.0.0</IdentityServer4EntityFrameworkStoragePackageVersion>
     <MessagePackPackageVersion>2.1.90</MessagePackPackageVersion>
+    <MicrosoftIdentityWebPackageVersion>0.2.0-preview</MicrosoftIdentityWebPackageVersion>
```

Comment:
Does this need to be rejected or is it acceptable given it's from a `5.0 release` to `master`?

Reply:
Keep this, it's an external dependency not managed by maestro/darc.